src/features/configure/vault/polygon_pools.js

```diff
@@ -543,6 +543,31 @@ export const polygonPools = [
     buyTokenUrl:
       'https://app.sushi.com/swap?inputCurrency=0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619&outputCurrency=0x61BDD9C7d4dF4Bf47A4508c0c8245505F2Af5b7b',
   },
+  {
+    id: 'sushi-wmatic-ghst',
+    name: 'WMATIC-GHST SLP',
```

Comment:
Switch these around so the interesting token is first (and on top in the images), so GHST-MATIC. And drop the W from MATIC in the name, token, and assets.

Reply:
> Did you also turn on harvestOnDeposit? If so then you can write the withdrawal fee as 0%.

I did not, wasn't sure this was something I should choose. What's the reasoning behind one or the other?